tests/ci/build-pkg.yml

```diff
@@ -38,6 +38,7 @@ steps:
   - |
     python3 -mvenv --system-site-packages build/venv/v
```

Comment:
minor: Since this is used only with python3.9+ should we check if `--upgrade-deps` does the job?
https://docs.python.org/3/library/venv.html -  https://bugs.python.org/issue34556

Reply:
oh, cool, didn't know about it. I'll add it